src/main/java/ua/danit/final_project/controllers/WorkShiftController.java

```diff
@@ -1,6 +1,8 @@
 package ua.danit.final_project.controllers;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.slf4j.LoggerFactory'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=169&issues=AWbzgf-NsSodgw_KP3jP&open=AWbzgf-NsSodgw_KP3jP)

[ISSUE_KEY]: AWbzgf-NsSodgw_KP3jP